dotcom-rendering/src/web/lib/buildNewsletterSignUpText.tsx

```diff
@@ -5,8 +5,9 @@ const specialCasedValues: Record<string, string> = {
 export const buildDetailText = (input: string) => {
 	const normalisedInput = input.toLowerCase().trim();
 
-	if (normalisedInput in specialCasedValues) {
-		return `Free ${specialCasedValues[normalisedInput]} newsletter`;
+	const specialCase = specialCasedValues[normalisedInput];
+	if (specialCase) {
+		return `Free ${specialCase} newsletter`;
```

Comment:
Needed to extract this to make the compiler happy.

@guardian/newsletters you might want to have a peek.

Reply:
looks good - thanks for flagging.